src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ListCaseConfirmationTest.java

```diff
@@ -33,41 +37,62 @@ class ListCaseConfirmationTest {
 
     private ListCaseConfirmation listCaseConfirmation = new ListCaseConfirmation();
 
-    @Test
-    void should_return_confirmation() {
+    @ParameterizedTest
+    @EnumSource(value = YesOrNo.class, names = { "NO", "YES" })
+    void should_return_confirmation(YesOrNo yesOrNo) {
```

Comment:
can you please change the name of the parameter as 'isAda'?

Reply:
I have changed the name of parameter yesOrNo to isAda